lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -31,11 +34,18 @@ const SortResultsDropdown = styled(Dropdown)`
   }
 `
 
+const UnicodeChevron = styled.div<{ direction: 'up' | 'down' }>`
+  transform: rotate(${(props) => (props.direction === 'up' ? '270' : '90')}deg)
+    scaleY(2) scaleX(1.5);
+`
```

Comment:
You're creating a boolean with strings here, I would just change the prop to something true/false!